### PR TITLE
Trip adv settings a11y

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
This PR contains accessibility improvements to the advanced trip settings pane.
The main change is around the trip monitoring start setting which I simplified the labeling, so we don't end up with nested labels as was the case before.

### CODE REVIEW

File: lib/components/user/monitored-trip/trip-notifications-pane.tsx [hunk 1]
```diff
@@ -233,53 +271,40 @@ class TripNotificationsPane extends Component {
...
-              aria-label={intl.formatMessage({
-                id: 'components.TripNotificationsPane.toggleAdvancedSettings'
-              })}
-              onClick={this._handleToggleAdvancedSettings}
-              type="button"
-            >
-              <StyledIconWrapper>
-                {showAdvancedSettings ? <CaretDown /> : <CaretRight />}
-              </StyledIconWrapper>
+          <details>
```
Comment (miles-grant-ibigroup) [inline lib/components/user/monitored-trip/trip-notifications-pane.tsx:276]: I believe it's this change that's making it difficult to see that advanced settings can be expanded! 

Screenshot 2023-02-17 at 10 16 57 AM

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Code looks good, just one minor display regression!

File: lib/components/user/monitored-trip/trip-notifications-pane.tsx (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/components/user/monitored-trip/trip-notifications-pane.tsx:276]: Is this on a particular browser?

**Comment by miles-grant-ibigroup** [comment]
Oh yes this is happening on Firefox

From: binh-dam-ibigroup ***@***.***>
Date: Friday, February 17, 2023 at 10:34 AM
To: opentripplanner/otp-react-redux ***@***.***>
Cc: Miles Grant ***@***.***>, Assign ***@***.***>
Subject: Re: [opentripplanner/otp-react-redux] Trip adv settings a11y (PR #795)
*** Exercise caution. This is an EXTERNAL email. DO NOT open attachments or click links from unknown senders or unexpected email. ***

@binh-dam-ibigroup commented on this pull request.

________________________________

In lib/components/user/monitored-trip/trip-notifications-pane.tsx<https://github.com/opentripplanner/otp-react-redux/pull/795#discussion_r1109986317>:

>

-          <h4>

-            <SettingsToggle

-              aria-expanded={showAdvancedSettings}

-              aria-label={intl.formatMessage({

-                id: 'components.TripNotificationsPane.toggleAdvancedSettings'

-              })}

-              onClick={this._handleToggleAdvancedSettings}

-              type="button"

-            >

-              <StyledIconWrapper>

-                {showAdvancedSettings ? <CaretDown /> : <CaretRight />}

-              </StyledIconWrapper>

+          <details>

Is this on a particular browser?

File: lib/components/user/monitored-trip/trip-notifications-pane.tsx (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/user/monitored-trip/trip-notifications-pane.tsx:276]: Oh yes my bad this is Firefox

Comment (binh-dam-ibigroup) [reply lib/components/user/monitored-trip/trip-notifications-pane.tsx:276]: Thanks for checking on Firefox! Fixed in https://github.com/opentripplanner/otp-react-redux/pull/795/commits/9e81ac57b0848a1f1772700f43a6318d1288488d and that should still work for Chromium and Safari.

**Review by miles-grant-ibigroup — APPROVED** [review]
Looking great on all browsers thanks for the changes! Percy also looking great -- better than dev haha

**Review by amy-corson-ibigroup — APPROVED** [review]

**Pull request merged**